core/src/datayoga_core/prom.py

```diff
@@ -0,0 +1,11 @@
+from prometheus_client import Counter, start_http_server
+
+incoming_records = Counter('incoming_records', 'Number of incoming records')
+
+processed_entries = Counter('processed_records', 'Number of processed records', ('step',))
+rejected_records = Counter('rejected_records', 'Number of rejected records', ('step',))
+filtered_records = Counter('filtered_records', 'Number of filtered records', ('step',))
```

Comment:
change ' to " to be consistent with the formatting we use

Reply:
✅